DEPLOYMENT.md

```diff
@@ -35,19 +56,182 @@ Our Docker images our tagged in four possible ways:
 Typically, we use *unstable* for testing and development and *latest* for production.
 We recommend you follow this if you are unsure which to use.
 
+## Installing Dependencies
+
+Update system packages: 
+`sudo apt update && sudo apt upgrade -y`
+
+Required software:
+`sudo apt install -y docker.io download docker`
+
+Verify: `docker --version`
+
+Install Docker Compose: `sudo apt install -y docker-compose`
+Verify: `docker-compose --version`
+
+Add user to docker group: (so you don't have to keep running with sudo):
+`sudo usermod -aG docker $USER`
+
+`newgrp docker`
+(Verify with `docker run hello-world`)
+
+Install NVIDIA drivers: `sudo apt install -y nvidia-driver nvidia-container-runtime`
+`sudo reboot`
+Verify: `nvidia-smi`
+
+
+
+Clone the repository
+
+ubuntu@ip-172-31-29-145:~$ `cd ~`
+
+ubuntu@ip-172-31-29-145:~$ `git clone --recurse-submodules git@github.com:Shared-Reality-Lab/IMAGE-server.git`
+
+Cloning into 'IMAGE-server'...
+The authenticity of host 'github.com (140.82.113.3)' can't be established.
+ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
+This key is not known by any other names
+Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
+
+If it fails or you see permission errors:
```

Comment:
For SSH issues, should we just point to documentation? This seems to take considerable space and makes this tutorial seem more imposing than is probably necessary?